AjaxControlToolkit.Jasmine/Global.asax.cs

```diff
@@ -10,6 +11,7 @@ public class Global : System.Web.HttpApplication {
             RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
 
+            AjaxFileUploadHelper.RootTempFolderPath = HttpContext.Current.Server.MapPath("~/Temp");
```

Comment:
In that case user will not be able to specify physical path like "C:\..." or "\\share".